site/source/support/faq/faq-general.rst

```diff
@@ -74,44 +74,40 @@ What are StartOS Services?
 --------------------------
 A Service can be any piece of software added to the Marketplace.  Unlike "apps," services are (usually) "server-side" software, meaning they are intended to run 24/7/365 and listen for requests from your clients (apps).  All services are "self-hosted," meaning that you are in complete control of your data.  This means you can run your own "cloud!"  Learn more about managing services :ref:`here <managing-services>` and see our currently `Available Services <https://marketplace.start9.com/>`_.
 
-Does the Embassy ship worldwide?
---------------------------------
-We ship everywhere that DHL ships.  Please consider that in some countries, the VAT and Customs fees are so ridiculous that they cost as much as Embassy itself or more.  You may wish to consider buying your hardware locally and building your own device, downloading StartOS from our github, and `donating <https://donate.start9.com/>`_ to us.  Please see the :ref:`DIY<diy>` page for details.
+Does Start ship worldwide?
+--------------------------
+We ship everywhere that DHL ships.  Please consider that in some countries, the VAT and Customs fees are so ridiculous that they cost as much as the server itself.  You may wish to consider buying your hardware locally and building your own device, downloading StartOS from our github, and `donating <https://donate.start9.com/>`_ to us.  Please see the :ref:`DIY<diy>` page for details.
 
-Does the Embassy have international electrical plugs?
------------------------------------------------------
+Do you have international electrical plugs?
+-------------------------------------------
 Power supplies for EU, AU, US, and UK are available.
 
-Is the power supply that comes with Embassy 220v compatible?
-------------------------------------------------------------
+Is the power supply 220v compatible?
```

Comment:
220v isn't standard anywhere - people would like search our docs for 230v.

Reply:
220V is the standard household voltage in dozens of countries, including China, Saudi Arabia, Chile, Egypt, the U.A.E, and South Korea.

Reply:
Maybe best to say that the PSUs are rated for 100-240V?  All 3 devices are labeled this way